insights/specs/insights_archive.py

```diff
@@ -235,6 +235,7 @@ class InsightsArchiveSpecs(Specs):
         "insights_commands/sudo_-iu_postgres_psql_-d_foreman_-c_select_count_from_hosts"
     ])
     satellite_custom_ca_chain = simple_file("insights_commands/awk_BEGIN_pipe_openssl_x509_-noout_-subject_-enddate_._-_BEGIN_CERT._._-_END_CERT._print_pipe_._-_END_CERT._close_pipe_printf_n_.etc.pki.katello.certs.katello-server-ca.crt")
+    satellite_enabled_features = simple_file("insights_commands/curl_-sk_https_..localhost_9090.features_--connect-timeout_5")
```

Comment:
@huali027 this definition is not necessary since this spec can only be collected with core collection due to the dependency `IsSatellite`.

Reply:
@bfahr, sure, I removed it, thanks for the review.